easybuild/easyblocks/i/impi.py

```diff
@@ -259,7 +259,8 @@ def make_module_req_guess(self):
         else:
             guesses = {}
             if LooseVersion(self.version) >= LooseVersion('2019'):
-                # Keep release_mt and release in front, to give them priority over the possible symlinks in intel64/lib.
+                # Keep release_mt and release in front, to give priority to the possible symlinks in intel64/lib.
+                # (remember paths are *prepended*, so the last path in the list has highest priority)
                 # IntelMPI 2019 changed the default library to be the non-mt version.
                 lib_dirs = ['intel64/%s' % x for x in ['lib/release_mt', 'lib/release', 'lib']]
```

Comment:
Once https://github.com/easybuilders/easybuild-framework/pull/3367 is merged and order is consistently reversed, this should be the order for `lib_dirs` (`release_mt` should be removed, only one of `release` or `release_mt` is needed)
```suggestion
                lib_dirs = ['intel64/%s' % x for x in ['lib', 'lib/release']]
```